idd/Energy+.idd.in

```diff
@@ -46768,6 +46768,20 @@ ZoneHVAC:EquipmentList,
        \minimum 0
        \note Specifies the zone equipment simulation order
        \note when the zone thermostat requests heating or no load
+  N3 , \field Zone Equipment 1 Sequential Cooling Fraction
+       \note The fraction of the remaining cooling load this equipment will attempt to serve
+       \note if the load distribution scheme is SequentialLoad, otherwise ignored.
+       \type real
+       \minimum 0.0
+       \maximum 1.0
+       \default 1.0
+  N4 , \field Zone Equipment 1 Sequential Heating or No-Load Fraction
```

Comment:
Calling this "Sequential Heating or No-Load Fraction" has nice symmetry with the sequence fields, but "No-Load Fraction" really makes no sense. If there is no load, we should't be touching any of the remaining load variables, I think. See below.

Reply:
I can make these changes.

Reply:
When you edit the I/O ref to match, there's a group heading about these field "Field Set (Zone Equipment: Object Type, Name, Cooling Sequence, Heating
or No-Load Sequence)", please add ". . . No-Load Sequence, *Sequential Cooling Fraction, Sequential Heating Fraction*)"

Reply:
And this will need some discussion in the Engineering Reference as well - but that can wait until after I/O freeze.